paper/src/main/java/com/denizenscript/denizen/paper/events/WardenChangesAngerLevelScriptEvent.java

```diff
@@ -0,0 +1,100 @@
+package com.denizenscript.denizen.paper.events;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import io.papermc.paper.event.entity.WardenAngerChangeEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class WardenChangesAngerLevelScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // warden changes anger level
+    //
+    // @Plugin Paper
+    //
+    // @Group Paper
+    //
+    // @Cancellable true
+    //
+    // @Location true
+    //
+    // @Triggers when a warden changes its anger level.
+    //
+    // @Player when the Entity who triggered the change is a player.
+    //
+    // @NPC when the Entity who triggered the change is a npc.
+    //
+    // @Context
+    // <context.entity> returns the EntityTag of the warden, which changed the anger level.
+    // <context.new_anger> returns an ElementTag of the new anger level.
+    // <context.old_anger> returns an ElementTag of the old anger level.
+    // <context.target> returns the EntityTag who triggered the change (if any).
+    //
+    // @Determine
+    // "ANGER:" + ElementTag(Number) to set the value of the anger level. Refer to <@link url https://jd.papermc.io/paper/1.19/org/bukkit/entity/Warden.AngerLevel.html>.

```

Comment:
This enum link being here is likely to be confusing (the enum isn't valid input, it's... I think just being linked to give a reference for what the number values mean?). It should be separated from the determination, and the purpose of the link clarified.